ledger-service/http-json-oracle/BUILD.bazel

```diff
@@ -30,7 +30,7 @@ da_scala_library(
         "@maven//:org_scalaz_scalaz_core",
```

Comment:
I will let @raphael-speyer-da to adapt and clean up everything he want in /ledger-service directory